src/app/Instance/ErrorHandling/ErrorHandlingSelection.tsx

```diff
@@ -68,14 +50,30 @@ export const ErrorHandlingSelection = ({
       isOpen={isSelectorOpen}
       selections={handlingMethod}
       menuAppendTo={"parent"}
+      isGrouped
     >
       {[
-        defaultMethodOption,
+        <SelectGroup
+          key="default-method-group"
+          className="default-method-group"
```

Comment:
This selector is a bit generic. Maybe we could tie it to the existing parent id. Like

```suggestion
          id="error-handling-method-selector__default-group"
```

Reply:
@kelvah done https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/234/commits/86d5326b37405d3e266e37ca018620b6376b7204